app/models/katello/repository.rb

```diff
@@ -261,6 +261,7 @@ def self.attribute_name
     end
 
     def set_container_repository_name
+      # TODO - fix this name render for container push
```

Comment:
@ianballou As of now this follows the LE pattern. That is one option, but the container_repository_name could also be this format: <org_label>/<product_label>/<container_name> (including capitals). If it doesn't matter at all I'd like to follow LE pattern since it's significantly easier to leave as-is.

Reply:
Following the LE pattern means the repository name in Katello and Pulp will differ. That means that users will end up pushing and pulling with different names, which would be different from normal container workflows.

For push, we should be able to completely skip the container repository name setting. One way to tell if a repository is a special push one could be via the version_href, since the version_href has that container-push string in it. Or, you could consider adding a tiny field in the database at creation time to denote that it's a container push repository.

I think it would be helpful to have a way to tell if a repo is a push one regardless -- we might need to lock down things in the UI, for example, which would mean the API would need to return if a repository is a container push one.

Reply:
`version_href` has been updated to match the pulp contents AND there is a `is_container_push` flag on the root repo. Should be all good to go.